backend/NEU-tts-2022/audio_handler.py

```diff
@@ -32,3 +33,17 @@ def AudioHandler():
     )
     response.headers['Access-Control-Allow-Origin'] = '*'
     return response
+
+
+@app.route('/get_phones', methods=['POST'])
+def Text2PhoneHandler():
+    kwargs = {}
+    request_dict = json.loads(request.get_data())
+    print(request_dict)
+    kwargs['text'] = request_dict.get('text')
+    kwargs['lex_path'] = request_dict.get('lex_path')
+
+    result = text2phones.get_phones(**kwargs)
```

Comment:
Hm, if arguments are optional here, I'm wondering if calling the method without arguments would work? Not sure.

Reply:
I think `lex_path` should be optional while `text` isn't, I'll change the code to reflect this. If `text` isn't present the `get` method will raise an error. Does it make sense?

Reply:
Oh `get` won't raise error if the key isn't present...I'll manually raise the error

Reply:
Updated in L44

Reply:
Ah, I see. Thanks for explaining!